lib/davinci_dtr_test_kit/client_groups/resp_assist_device/dtr_resp_questionnaire_package_request_test.rb

```diff
@@ -38,8 +38,7 @@ class DTRRespQuestionnairePackageRequestTest < Inferno::Test
             Launch. These references help determine the behavior of the app. Referenced instances
             may be provided in the "EHR-available resources" input.
           ),
-          default: JSON.pretty_generate([{ reference: 'Coverage/cov015' },
-                                         { reference: 'DeviceRequest/devreqe0470' }])
+          default: JSON.pretty_generate(['Coverage/cov015', 'DeviceRequest/devreqe0470'])
```

Comment:
Looks like this fhirContext format is a difference between [SMART 2.0](https://hl7.org/fhir/smart-app-launch/STU2/scopes-and-launch-context.html#launch-context-arrives-with-your-access_token) and [SMART 2.2](https://hl7.org/fhir/smart-app-launch/STU2.2/scopes-and-launch-context.html#launch-context-arrives-with-your-access_token). I don't think we want to change the default at this time, but we might want to change it based on a suite option around which version of SMART App Launch is in use. But I think we should leave that for a future ticket.

Reply:
Reverted https://github.com/inferno-framework/davinci-dtr-test-kit/pull/24/commits/fdb786435668713d0718e15e8c1444925e4e36be